tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -4,7 +4,7 @@
 Executable: EvolveGhSingleBlackHole
 Testing:
   Check: parse;execute_check_output
-  Timeout: 8
+  Timeout: 30
```

Comment:
Do you still need this long of a timeout now that you decreased the slab size?

Reply:
ah no, after looking at how long it took the tests in github, it seems 15 seconds should be good